tests/test_api_client.py

```diff
@@ -12,6 +12,38 @@
 class TestRossumClient:
     rossum_client = RossumClient(None)
 
+    @pytest.mark.usefixtures("mock_login_request")
+    def test_create_hook_extension_source(self, requests_mock):
+        requests_mock.post(
+            HOOKS_URL,
+            json={"id": 100200},
+            status_code=201,
+        )
+
+        args = ["name1", "hook_type1", [24412]]
+        kwargs = dict(
+            active=True,
+            events=["annotation_content.initialize", "annotation_content.user_update"],
+            sideload=[],
+            config={
+                "url": "httpmock:/adfa.asdf",
+                "secret": "sdf",
+                "insecure_ssl": False,
+            },
+            token_owner="httpmock://tokenowner.url",
+            extension_source="rossum_store",
+        )
+        self.rossum_client.create_hook(*args, **kwargs)
+        assert requests_mock.last_request.json()["extension_source"] == "rossum_store"
```

Comment:
Consider using `additional_matcher` parameter in the POST request on HOOKS_URL to check what you are sending instead of all these rows - it will help make the code cleaner.